src/global.ts

```diff
@@ -45,6 +45,7 @@ declare global {
   var IMTRPC: typeof publicApi.IMTRPC;
   var IMTFeeder: typeof publicApi.IMTFeeder;
   var IMTTransformer: typeof publicApi.IMTTransformer;
+  var IMTHITL: typeof publicApi.IMTHITL;
```

Comment:
This was for some reason missing, but the blueprint has this line of code that depends on this in global:

Seems like we forgot to add it when doing some refactoring
```
       if (klass.prototype instanceof IMTHITL)
```